docs/Contributors_Guide/github_workflow.rst

```diff
@@ -10,24 +10,26 @@ How Unified Workflow releases are created
 
 The branching model employed by the workflow-tools GitHub repository is similar to
 that described in
-`A successful Git branching model <https://nvie.com/posts/a-successful-git-branching-model/>`_,
+`A successful Git branching model <https://nvie.com/posts/a-successful-git-branching-model/>`__,
 where new or updated code is created on a 'feature' branch that is based on
-the `workflow-tools GitHub 'develop' branch <https://github.com/ufs-community/workflow-tools/tree/develop>`_.
+the `workflow-tools GitHub 'develop' branch <https://github.com/ufs-community/workflow-tools/tree/develop>`__.
+Developers should create an issue for each feature they plan to work on. 
 
-The feature branch is named after the corresponding Jira issue:
+The feature branch is named after the corresponding GitHub issue:
```

Comment:
We had actually been using the Jira ticket. @fgabelmannjr is that right, or do we want this to be a GitHub Issue number. I'm indifferent, and honestly don't care what branches are named.... 😬 

For a Community-based contribution, though, I see where this makes total sense. They don't have access to the Jira board.

Reply:
I would agree with GitHub issue, plus its a GitHub workflow